src/Workspaces/Remote/ServiceHub/Services/ServiceBase.cs

```diff
@@ -13,6 +13,8 @@
 using Newtonsoft.Json.Linq;
 using Roslyn.Utilities;
 
+#pragma warning disable CS0618 // Type or member is obsolete - this should become error once we provide infra for migrating to ISB (https://github.com/dotnet/roslyn/issues/44326)
```

Comment:
❔ Why disable instead of marking type or members obsolete?

Reply:
So that we don't report the diagnostic in partner's repo since we don't have replacement yet.